src/components/Pagination.tsx

```diff
@@ -0,0 +1,79 @@
+import { Dispatch, SetStateAction, useMemo } from "react";
+import { IconButton } from "src/components";
+import { Css, Palette } from "src/Css";
+import { SelectField } from "src/inputs";
+import { useTestIds } from "src/utils";
+
+export type PageSettings = {
+  page: number;
+  perPage: number;
+};
+
+type PaginationProps = {
```

Comment:
**request:** To be consistent with our other components, can we make this an `interface` instead of a type?

Reply:
Done 👍